### DISCUSSION TITLE
Supported external Download Manager

### DISCUSSION THREAD

**Discussion by LittleFreak** [aniyomiorg/aniyomi#1441] [Q&A]
The settings implying, that some download manager are supported by default. (`none`) #985
Which would that be and which would you actually recommend?
I've tried seal and a few others from f-droid so far, but all lacked the feature to download to its destined path for aniyomi.

**Comment by AleJavierHY** [comment]
I recommend **1DM**, I am using it and I have never had any problems

**Reply by LittleFreak** [reply]
> I recommend **1DM**, I am using it and I have never had any problems

Why 1DM? Will every episode be downloaded to its respective aniyomi folder or are they all together in `sdcard/downloads` for example?

**Reply by AleJavierHY** [reply]
1DM allows you to modify the download path of each chapter before adding it to your download list, you just have to show it the correct address within the Aniyomi folder to be able to play the chapter from the application.

**Reply by LittleFreak** [reply]
> .

How about batch downloads of the same Anime/library entry then?

**Reply by AleJavierHY** [reply]
Unfortunately 1DM shows you a small floating window for each chapter of the series and in each one you must tell it where to save the chapter within the Aniyomi directory.
For those cases, what I do is download it in Download within the internal memory of the phone and then I move them to their respective folder within Aniyomi (when using the external download manager, Aniyomi creates the chapter folder and a file with 0B in its download directory, you must overwrite it)
I hope it helps you, if you find a better method I would appreciate it if you would share the information

**Reply by NUTTERAT** [reply]
how do you overwrite it?
